qp_qiime2/qp_qiime2.py

```diff
@@ -18,6 +18,10 @@
 import qiime2
 import pandas as pd
 
+# avoid some alpha metrics within the alpha_rarefaction method
+from q2_diversity._alpha._visualizer import (
```

Comment:
Looks good, although the metrics are being imported from a private module i.e. `_alpha` and `_visualizer`. Maybe it would make sense to change that in q2_diversity so this set is exposed from q2_diversity? Should help ensuring that this isn't removed/renamed down the road.

Side note, based on the GitHub repo, this can be imported from `_alpha`:

```suggestion
from q2_diversity._alpha import (
```

https://github.com/qiime2/q2-diversity/blob/master/q2_diversity/_alpha/__init__.py

Reply:
I see your point but, in all fairness, if it changes in q2_diversity that change will be documented in the changelog and we will detect here too so not sure if needed ...

Reply:
Hmm I guess my point is that since this is not a public-facing interface it shouldn't be expected that these updates are reflected in the Changelog. Perhaps they easiest would be to merge this as is and submit a PR to q2_diversity where this attribute is made public i.e. include it in the top-level __init__.py file.